pip install ninja will download googletest from GitHub

```
410.6               warnings.warn("unprocessed git archival found (no export subst applied)")
410.6             *** scikit-build-core 0.11.6 using CMake 3.31.7 (wheel)
410.6             *** Configuring CMake...
410.6             loading initial cache file build/py3-none-linux_x86_64/CMakeInit.txt
410.6             -- The C compiler identification is GNU 14.2.0
410.6             -- The CXX compiler identification is GNU 14.2.0
410.6             -- Detecting C compiler ABI info
410.6             -- Detecting C compiler ABI info - done
410.6             -- Check for working C compiler: /usr/bin/gcc - skipped
410.6             -- Detecting C compile features
410.6             -- Detecting C compile features - done
410.6             -- Detecting CXX compiler ABI info
410.6             -- Detecting CXX compiler ABI info - done
410.6             -- Check for working CXX compiler: /usr/bin/g++ - skipped
410.6             -- Detecting CXX compile features
410.6             -- Detecting CXX compile features - done
410.6             -- ************************************
410.6             -- Ninja Python Distribution
410.6             --
410.6             --   RUN_NINJA_TEST : OFF
410.6             -- ************************************
410.6             -- IPO / LTO enabled
410.6             -- Performing Test flag_no_deprecated
410.6             -- Performing Test flag_no_deprecated - Success
410.6             -- Looking for ppoll
410.6             -- Looking for ppoll - found
410.6             CMake Warning at ninja-upstream/CMakeLists.txt:82 (message):
410.6               re2c 2 or later was not found; changes to src/*.in.cc will not affect your
410.6               build.
410.6       
410.6       
410.6             -- Looking for fork
410.6             -- Looking for fork - found
410.6             -- Looking for pipe
410.6             -- Looking for pipe - found
410.6             -- Could NOT find GTest (missing: GTEST_LIBRARY GTEST_INCLUDE_DIR GTEST_MAIN_LIBRARY)
410.6             [ 11%] Creating directories for 'googletest-populate'
410.6             [ 22%] Performing download step (download, verify and extract) for 'googletest-populate'
410.6             -- Downloading...
410.6                dst='/tmp/pip-install-qv4moum4/ninja_17654afe141548b69f95e4512222c44e/build/py3-none-linux_x86_64/_deps/googletest-subbuild/googletest-populate-prefix/src/release-1.12.1.tar.gz'
410.6                timeout='none'
410.6                inactivity timeout='none'
410.6             -- Using src='https://github.com/google/googletest/archive/refs/tags/release-1.12.1.tar.gz'
410.6             CMake Error at googletest-subbuild/googletest-populate-prefix/src/googletest-populate-stamp/download-googletest-populate.cmake:163 (message):
410.6               Each download failed!
410.6       
410.6                 error: downloading 'https://github.com/google/googletest/archive/refs/tags/release-1.12.1.tar.gz' failed
410.6                       status_code: 56
410.6                       status_string: "Failure when receiving data from the peer"
410.6                       log:
410.6                       --- LOG BEGIN ---
410.6                       timeout on name lookup is not supported
410.6       
410.6               Host github.com:443 was resolved.
410.6       
410.6               IPv6: (none)
410.6       
410.6               IPv4: 20.205.243.166
410.6       
410.6                 Trying 20.205.243.166:443...
410.6       
410.6               ALPN: curl offers h2,http/1.1
410.6       
410.6               [5 bytes data]
410.6       
410.6               TLSv1.3 (OUT), TLS handshake, Client hello (1):
410.6       
410.6               [1556 bytes data]
410.6       
410.6                CAfile: /etc/ssl/certs/ca-certificates.crt
410.6                CApath: /etc/ssl/certs
410.6       
410.6               [5 bytes data]
410.6       
410.6               TLSv1.3 (IN), TLS handshake, Server hello (2):
410.6       
410.6               [122 bytes data]
410.6       
410.6               [5 bytes data]
410.6       
410.6               TLSv1.3 (IN), TLS change cipher, Change cipher spec (1):
410.6       
410.6               [1 bytes data]
410.6       
410.6               [5 bytes data]
410.6       
410.6               [1 bytes data]
410.6       
410.6               TLSv1.3 (IN), TLS handshake, Encrypted Extensions (8):
410.6       
410.6               [19 bytes data]
410.6       
410.6               [5 bytes data]
410.6       
410.6               [1 bytes data]
410.6       
410.6               TLSv1.3 (IN), TLS handshake, Certificate (11):
410.6       
410.6               [3135 bytes data]
410.6       
410.6               [5 bytes data]
410.6       
410.6               [1 bytes data]
410.6       
410.6               TLSv1.3 (IN), TLS handshake, CERT verify (15):
410.6       
410.6               [79 bytes data]
410.6       
410.6               [5 bytes data]
410.6       
410.6               [1 bytes data]
410.6       
410.6               TLSv1.3 (IN), TLS handshake, Finished (20):
410.6       
410.6               [36 bytes data]
410.6       
410.6               [5 bytes data]
410.6       
410.6               TLSv1.3 (OUT), TLS change cipher, Change cipher spec (1):
410.6       
410.6               [1 bytes data]
410.6       
410.6               [5 bytes data]
410.6       
410.6               [1 bytes data]
410.6       
410.6               TLSv1.3 (OUT), TLS handshake, Finished (20):
410.6       
410.6               [36 bytes data]
410.6       
410.6               SSL connection using TLSv1.3 / TLS_AES_128_GCM_SHA256 / x25519 /
410.6               id-ecPublicKey
410.6       
410.6               ALPN: server accepted h2
410.6       
410.6               Server certificate:
410.6       
410.6                subject: CN=github.com
410.6                start date: Feb  5 00:00:00 2025 GMT
410.6                expire date: Feb  5 23:59:59 2026 GMT
410.6                subjectAltName: host "github.com" matched cert's "github.com"
410.6                issuer: C=GB; ST=Greater Manchester; L=Salford; O=Sectigo Limited; CN=Sectigo ECC Domain Validation Secure Server CA
410.6                SSL certificate verify ok.
410.6                 Certificate level 0: Public key type EC/prime256v1 (256/128 Bits/secBits), signed using ecdsa-with-SHA256
410.6                 Certificate level 1: Public key type EC/prime256v1 (256/128 Bits/secBits), signed using ecdsa-with-SHA384
410.6                 Certificate level 2: Public key type EC/secp384r1 (384/192 Bits/secBits), signed using ecdsa-with-SHA384
410.6       
410.6               [5 bytes data]
410.6       
410.6               [1 bytes data]
410.6       
410.6               Connected to github.com (20.205.243.166) port 443
410.6       
410.6               using HTTP/2
410.6       
410.6               [HTTP/2] [1] OPENED stream for
410.6               https://github.com/google/googletest/archive/refs/tags/release-1.12.1.tar.gz
410.6       
410.6       
410.6               [HTTP/2] [1] [:method: GET]
410.6       
410.6               [HTTP/2] [1] [:scheme: https]
410.6       
410.6               [HTTP/2] [1] [:authority: github.com]
410.6       
410.6               [HTTP/2] [1] [:path:
410.6               /google/googletest/archive/refs/tags/release-1.12.1.tar.gz]
410.6       
410.6               [HTTP/2] [1] [user-agent: curl/8.10.1]
410.6       
410.6               [HTTP/2] [1] [accept: */*]
410.6       
410.6               [5 bytes data]
410.6       
410.6               [1 bytes data]
410.6       
410.6               GET /google/googletest/archive/refs/tags/release-1.12.1.tar.gz HTTP/2
410.6       
410.6               Host: github.com
410.6       
410.6               User-Agent: curl/8.10.1
410.6       
410.6               Accept: */*
410.6       
410.6       
410.6       
410.6               Request completely sent off
410.6       
410.6               OpenSSL SSL_read: SSL_ERROR_SYSCALL, errno 0
410.6       
410.6               Failed receiving HTTP2 data: 56(Failure when receiving data from the peer)
410.6       
410.6               Connection #0 to host github.com left intact
```

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -175,7 +175,7 @@ jobs:
           done
 
       - name: Install SDist
-        run: .venv/bin/pip install --no-binary=ninja $(ls sdist/*.tar.gz)
+        run: .venv/bin/pip install -v --no-binary=ninja $(ls sdist/*.tar.gz)
 
       - name: Test installed SDist
         run: .venv/bin/pip check && .venv/bin/pytest ./tests

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -185,7 +185,7 @@ jobs:
           done
 
       - name: Install SDist
-        run: .venv/bin/pip install --no-binary=ninja $(ls sdist/*.tar.gz)
+        run: .venv/bin/pip install -v --no-binary=ninja $(ls sdist/*.tar.gz)
 
       - name: Test installed SDist
         run: .venv/bin/pip check && .venv/bin/pytest ./tests

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -23,6 +23,8 @@ message(STATUS "")
 message(STATUS "  RUN_NINJA_TEST : ${RUN_NINJA_TEST}")
 message(STATUS "************************************")
 
+set(BUILD_TESTING ${RUN_NINJA_TEST} CACHE INTERNAL "")
+
 add_subdirectory(ninja-upstream)
 
 if(RUN_NINJA_TEST)

